web3/_utils/method_formatters.py

```diff
@@ -315,6 +315,25 @@ def apply_list_to_array_formatter(formatter: Any) -> Callable[..., Any]:
 )
 
 
+call_without_override: Callable[
+    [Tuple[Dict[str, Any], Union[str, int]]],
+    Tuple[Dict[str, Any], int]
+]
+call_without_override = apply_formatters_to_sequence([
+    transaction_param_formatter,
+    to_hex_if_integer,
+])
+call_with_override: Callable[
+    [Tuple[Dict[str, Any], Union[str, int], Dict[str, Any]]],
+    Tuple[Dict[str, Any], int, Dict[str, Any]],
+]
+call_with_override = apply_formatters_to_sequence([
+    transaction_param_formatter,
+    to_hex_if_integer,
+    lambda x: x,
```

Comment:
I don't think it really matters in this case, but you could import `identity` from `eth_utils.tools` and use it here 🤷

Reply:
TIL `identity`. first impression is that i'm not a big fan of bringing it in here, since its an extra layer of indirection, and unused elsewhere in the formatters. not a strongly held opinion, though.